### ISSUE TITLE
question, not issue, should I use all training data for embedding?

### ISSUE THREAD

**Issue by superfan123** [entron/entity-embedding-rossmann#25]
Should I use all training data for embedding and then attach the embedded feature to train/test and then train XGB?
Or I need to split the training into two set and learn the embedding from on set and then use the other set to train XGB?

**Comment by liaoyuhua** [comment]
Of course you should use all training data for embedding layer. Generally, more data means more information, as long as you avoid to obtain information from test set.

**Comment by superfan123** [comment]
Thanks.
